### DISCUSSION TITLE
Teleportation when entering a region

### DISCUSSION THREAD

**Discussion by Ophise** [Pryaxis/TShock#2184] [Q&A]
Hello! Is there a plugin that teleports players entering a specific region?
_P.S. Teleport the player to the specified coordinates_

**Comment by Kojirremer** [comment]
If it is specifically when entering a region no matter what, you can configure Smart Regions to do that. If it doesn't matter if it is a region as long as you teleport when stepping on a specific place, then Advanced Warpplates can do it as well.
